spec/features/home_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+describe "links tests" do
+  it "opens add_a_new_card_page, after clicking on link 'Добавить карточку'" do
+    visit '/'
+    click_link("Добавить карточку")
+    expect(page).to have_content "Новая карточка"
+  end
+  
+  it "opens page containing all cards, after clicking on link 'Все карточки'" do
+    visit '/'
+    click_link("Все карточки")
+    page.has_selector?("all-cards-list")
+  end
+end
+
+describe "the card selection process" do
+  before :each do
+    @test_card = Card.create(original_text: "Правильное значение",
+                           translated_text: "Correct value",
+                           review_date: Date.tomorrow)   
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.